### PR TITLE
add docs for bookings_fees

### PR DESCRIPTION
**Pull request by kaizencodes**
for [3185](https://github.com/BookingSync/BookingSync/pull/3185)

### CODE REVIEW

**Comment by kaizencodes** [comment]
@Azdaroth done

File: content/reference/changelog.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## 2017-10-02
+  * [api improvement] Add `add_bookings_fees` endpoint
+  * [api improvement] Add `remove_bookings_fees` endpoint
```
Comment (Azdaroth) [inline content/reference/changelog.md]: Why duplicating? :P

Comment (kaizencodes) [reply content/reference/changelog.md]: @Azdaroth Seemed like the convention, I can merge it if you'd like

Comment (Azdaroth) [reply content/reference/changelog.md]: Wow, is it really duplicated with the same case for 2 different labels?

Comment (Azdaroth) [reply content/reference/changelog.md]: @kaizencodes ^

Comment (kaizencodes) [reply content/reference/changelog.md]: @Azdaroth done

**Comment by Azdaroth** [comment]
@kaizencodes Can you please merge and release it?

**Pull request merged**